development/openstack-base-bionic-rocky/README.md

```diff
@@ -0,0 +1,186 @@
+# Basic OpenStack Cloud
+
+*DEV/TEST ONLY*: This unstable, development example bundle deploys a basic OpenStack Cloud providing Dashboard, Compute, Network, Block Storage, Object Storage, Identity and Image services. See also: [Stable Bundles](https://jujucharms.com/u/openstack-charmers).
+
+This example bundle deploys a basic OpenStack Cloud (Rocky with Ceph Mimic) on Ubuntu 18.04 LTS (Bionic), providing Dashboard, Compute, Network, Block Storage, Object Storage, Identity and Image services.
+
+## Requirements
+
+This example bundle is designed to run on bare metal using Juju 2.x with [MAAS][] (Metal-as-a-Service); you will need to have setup a [MAAS][] deployment with a minimum of 4 physical servers prior to using this bundle.
+
+Certain configuration options within the bundle may need to be adjusted prior to deployment to fit your particular set of hardware. For example, network device names and block device names can vary, and passwords should be yours.
+
+Servers should have:
+
+ - A minimum of 8GB of physical RAM.
+ - Enough CPU cores to support your capacity requirements.
+ - Two disks (identified by /dev/sda and /dev/sdb); the first is used by MAAS for the OS install, the second for Ceph storage.
+ - Two cabled network ports on eno1 and eno2 (see below).
+
+Servers should have two physical network ports cabled; the first is used for general communication between services in the Cloud, the second is used for 'public' network traffic to and from instances (North/South traffic) running within the Cloud.
+
+## Components
+
+ - 1 Node for Neutron Gateway and Ceph with RabbitMQ and MySQL under LXC containers.
```

Comment:
This says that ceph should be on the neutron gateway node as well, while the bundle, below, does not place ceph on that node

Reply:
Strictly speaking it does place Ceph, to be specific the Ceph RADOS Gateway, on the Neutron gateway node. :-P